test/integration/organizations_test.rb

```diff
@@ -0,0 +1,62 @@
+require "test_helper"
+
+class OrganizationsTest < ActionDispatch::IntegrationTest
+  setup do
+    @user = create(:user, remember_token_expires_at: Gemcutter::REMEMBER_FOR.from_now)
+    post session_path(session: { who: @user.handle, password: PasswordHelpers::SECURE_TEST_PASSWORD })
+  end
+
+  test "should show an organization" do
+    organization = create(:organization, owners: [@user], handle: "arrakis", name: "Arrakis")
+    organization.rubygems << create(:rubygem, name: "arrakis", number: "1.0.0")
+
+    get "/organizations/#{organization.to_param}"
```

Comment:
Can use we use the rails `organizations_path(organization)`, including below and other test files in this PR.

Reply:
Done, moved to `test/functional` which more correctly fits this pattern.